cobalt/build/archive_test_artifacts.py

```diff
@@ -101,41 +98,36 @@ def create_archive(
         if any(line.startswith(path) for path in _EXCLUDE_DIRS):
           continue
 
-        if is_android and line.startswith('../../'):
+        if root_dir and line.startswith('../../'):
           target_src_root_deps.add(line[6:])
         else:
           rel_path = os.path.relpath(os.path.join(tar_root, line.strip()))
           target_deps.add(rel_path)
       combined_deps |= target_deps
 
-      # Android tests and deps are bundled into one tar file per target.
-      if is_android:
+      if archive_per_target:
         output_path = os.path.join(destination_dir,
                                    f'{target_name}_deps.tar.gz')
-        _make_tar(output_path, [(combined_deps, out_dir),
-                                (target_src_root_deps, source_dir)])
+
+        # TODO: Something is fishy here.
```

Comment:
Still fishy?

Reply:
No, I think I was just looking at the wrong diff. This makes sense.